docs/titan-reference/button-reference.md

```diff
@@ -0,0 +1,485 @@
+
+ Example: \<Copy\> {Source Playback} {Destination Playback}
+
+ Buttons | Action
+ --------|--------
+ \<Copy\> \<Copy\> | Same as pressing \<Move\>
+ 
+ --- 
+## \<Cue\> 
+ Used when selecting cues in Cue Lists. Labelled Connect on some consoles, see [Connect](#Connect).
+
+ --- 
+## \<Cue +\> 
+ (D9 only) Moves the next cue in the connected cue list on by one - same as \<Next Step\>.
+
+ --- 
+## \<Cue -\> 
+ (D9 only) Moves the next cue in the connected cue list back by one - same as \<Prev Step\>.
+
+ --- 
+## \<Custom\> 
+ (D9 only) Not used yet.
+
+ --- 
+## \<Delete\> 
+ Used to delete items. You usually need to press the item to be deleted a second time to confirm the delete, or press \<Enter\> to confirm. Use \<Latch\> to keep the Delete function active for deleting multiple items.
+
+ Example: \<Delete\> {Playback} {Playback}
+
+ --- 
+## \<Direction\> 
+ (D9 only) Used when setting direction of shapes. See [Shape Direction](../effects/shape-generator.md/#shape-direction)
+
+ --- 
+## \<Disk\> 
+ Shows disk functions such as Save Show etc. If pressed when \<Avo\> held, enters System menu. See [Saving the show](../titan-basics/loading-and-saving-shows.md#saving-the-show) and [The System Menu](../system-settings/the-system-menu.md).
+
+ --- 
+## \<Edit\> 
+ Used when editing palettes. See [Changing the content of palettes](../palettes/editing-palettes.md/#changing-the-content-of-palettes).
+
+ --- 
+## \<Effect\> 
+ Selects the Effect attribute group so the wheels control Prism etc. Also used for creating masks. See [Changing Attributes using the Wheels](../controlling-fixtures/changing-fixture-attributes.md/#changing-attributes-using-the-wheels).
+
+ --- 
+## \<Enter\> 
+ Completes or accepts many commands.
+
+ --- 
+## \<Enter/B\> 
+ (D9 only) Enters Scene Master preset mode. See [Scene Master](../running-the-show/playback-controls.md/#scene-master).
+
+ --- 
+## \<Exit\> 
+ Leaves the current menu and goes one level back up the menu structure.
+
+  Buttons | Action
+ --------|--------
+  \<Avo\> + \<Exit\> | Exit menu structure back to the top level
+
+ --- 
+## \<Exit/A\> 
+ (D9 only) Exits Scene Master preset mode and goes back to Live. See [Scene Master](../running-the-show/playback-controls.md/#scene-master).
+
+ --- 
+## \<FX\> 
+ (D9 only) Selects the Shape/FX menu to run shapes, key frame shapes or pixel map effects. Same as \<Shape\> on other consoles. See [Shape](#Shape).
+
+ --- 
+## \<FX (wheels)\> 
+ (D9 only - button to the right of the wheels) Sets the wheels into FX mode to control shape size, speed, spread.
+
+ --- 
+## \<Fade/Delay\> 
+ (D9 only) Sets the wheels into Fade/Delay mode to control fade and delay times.
+
+ --- 
+## \<Fan\> 
+ Starts Fan mode so the wheels spread out attributes rather than setting values. See [Fan Mode](../controlling-fixtures/changing-fixture-attributes.md/#fan-mode).
+
+ --- 
+## \<Fix +1\> 
```

Comment:
I think people will get it from that - not sure it is worth putting extra text to explain that

Reply:
Fair, I was more thinking that perhaps it should be listed as `<Fix +>` rather than `<Fix +1>` as that's the "new" name for the button. Doesn't really matter though I guess